### PR TITLE
Ten Green Bottles

### PR DESCRIPTION
**Pull request by habere-et-dispertire**
A renaming of the Beer Song.
See:  [[Suggestion] Rename the Beer Song to Ten Green Bottles #2066](https://github.com/exercism/problem-specifications/issues/2066)

### CODE REVIEW

**Comment by habere-et-dispertire** [comment]
I based this off the three files I found in the Beer Song exercise. I used fresh UUIDs (from configlet).

Is there anything else that needs doing? Thanks. :)

**Review by kytrinyx — APPROVED** [review]
This looks fine to me. I'll let someone who is familiar with the current state of this repo do the merging, though, in case I've missed something since the last time I touched it.

**Review by SleeplessByte — APPROVED** [review]
I also believe this is all right, except for the markdown to be incorrectly formatted. See [https://exercism.org/docs/building/markdown/markdown#h-layout](https://exercism.org/docs/building/markdown/markdown#h-layout). 

I know the old exercise is not yet in this format, but new contributions should.

**Review by ErikSchierboom — COMMENTED** [review]
Do we want to deprecate the old `beer-song` exercise? If so, we should add a `.deprecated` file to it, similar to https://github.com/exercism/problem-specifications/blob/main/exercises/binary/.deprecated

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
Maybe a sanity check from @exercism/reviewers?

File: exercises/bottle-song/.deprecated [hunk 1]
```diff
@@ -0,0 +1,3 @@
+**NOTE: This exercise has been deprecated**
```
Comment (kotp) [inline exercises/bottle-song/.deprecated:1]: Has this been deprecated? I thought this was bottle-song, which it refers to using instead on the very next line.

**Review by kotp — CHANGES_REQUESTED** [review]
Wrong deprecation note, or at least, I think, in the wrong location.

File: exercises/bottle-song/.deprecated (hunk 1 above)
Comment (habere-et-dispertire) [reply exercises/bottle-song/.deprecated:1]: Well spotted. So shall I move this to the beer-song folder?

Comment (kotp) [reply exercises/bottle-song/.deprecated]: Yes, I believe that would be the right location.

Comment (habere-et-dispertire) [reply exercises/bottle-song/.deprecated]: I moved it from the bottle-song folder to the beer-song folder.

**Review by kotp — APPROVED** [review]

**Review by SleeplessByte — APPROVED** [review]
Checked the changes since my last review. I think this is good to go, so let's get this merged. Thanks @habere-et-dispertire !

**Comment by SleeplessByte** [comment]
@iHiD perhaps want a similar icon, should be easy for Taiy.

**Comment by habere-et-dispertire** [comment]
I thank each and every one of you for your help. Much appreciated! :)

**Pull request merged**